src/pages/components/number-input/accessibility.mdx

```diff
@@ -19,99 +19,91 @@ import {
+
+<Row>
+<Column colLg={8}>
+
+![example of number input keyboard interaction](images/number-input-accessibility-2.png)
+
+<Caption>
+  Messages are surfaced to assistive technologies, and color alone is not used to visually signal errors.
+</Caption>
+
+</Column>
+</Row>
+
+## Design recommendations
+
+Design annotations are needed for the following instance.
+
+### Cue users for value and step
+Carbon offers the ability for the author to set minimum and maximum values for the input. A step value can also be set, which causes the value to increment by any amount. When setting limits on number entry, designers should warn the user in advance, instead of users discovering limits through an error message. This is particularly important for step values other than 1 (which should be avoided, where possible), as if a user's typed value does not match the step increment, it will be disallowed for no apparent reason.
 
 <Row>
-  <Column noGutterSm>
-    <StructuredListWrapper>
-      <StructuredListHead>
-        <StructuredListRow head>
-          <StructuredListCell head>
-            Automated test environment
-          </StructuredListCell>
-          <StructuredListCell head>Results</StructuredListCell>
-        </StructuredListRow>
-      </StructuredListHead>
-      <StructuredListBody>
-        <StructuredListRow>
-          <StructuredListCell>
-            - macOS Mojave version 10.14.2 with VoiceOver
-            <br />
-            - Chrome version 77.0.3865.90
-            <br />
-            - Dynamic Assessment Plugin (DAP) version 1.8.0.0 - IBM
-            Accessibility WCAG 2.1 Sept. 2019 Ruleset
-            <br />- Carbon React version 7.7.1
-          </StructuredListCell>
-          <StructuredListCell>
-            <strong>DAP test:</strong>
-            <br />- No violations
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  </Column>
+<Column colLg={8}>
+
+![a numeric input with helper text to enter values between 30 and 70, in increments of 5](images/number-input-accessibility-3.png)
+
+<Caption>
+  Notify users of allowable input ranges through helper text.
+</Caption>
+
+</Column>
 </Row>
+
+### Reduce the width of the number input
+
+Carbon’s default number input component is the same width as the default text input, yet numbers typically take up much less space. The + and - buttons are the primary user cue for a number input; however, people with low vision who use magnification may not see this cue if the buttons and label are not visible in their viewport at the same time. Where possible, reduce the width of the number field to improve its accessibility.
+
+<Row>
+<Column colLg={8}>
+
+![maybe a do don't pair?](images/number-input-accessibility-4.png)
```

Comment:
You've got a broken image here. 
<img width="811" alt="image" src="https://user-images.githubusercontent.com/11670886/178062010-66176cda-4589-496b-92e2-b26e93493a55.png">